stubs/protobuf/google/protobuf/message.pyi

```diff
@@ -31,9 +31,6 @@ class Message:
     def ByteSize(self) -> int: ...
     @classmethod
     def FromString(cls: type[Self], s: bytes) -> Self: ...
-    # The TypeVar must be bound to `Message` or we get mypy errors, so we cannot use `Self` for `Extensions`
-    @property
-    def Extensions(self: _M) -> _ExtensionDict[_M]: ...
```

Comment:
Hi all, my CI starts failing because of some my code trying to get the extensions:
```py
extensions = field_descriptor.GetOptions().Extensions
```

Where did that go? Is it invalid to get the Extensions that way?

Reply:
Fix is in https://github.com/python/typeshed/pull/7154